### DISCUSSION TITLE
Initial imposed displacements

### DISCUSSION THREAD

**Discussion by victorkemp** [OpenRadioss/OpenRadioss#3161] [Q&A]
I want to model a spring that's initially compressed and stationary, then released to bounce around. Since OpenRadioss doesn't have initial conditions for displacement, is the proper way to put it in that "initial" state using imposed displacement (non-zero) and velocity (zero) for a short time then release both of those at some small time value (Tstop > 0)? This seems a bit complicated so I wonder if I'm missing something.

**Comment by SyazwanSamad** [comment]
If I want to do a pre compressed spring, I would probably define it as non linear spring with positive value of displacement at force = 0 (and then define the rest of the curve using the intended slope/stiffness)

**Comment by MarianBulla** [comment]
Hi @victorkemp  and @SyazwanSamad,
for pre-compressed spring, we already have an example. Exactly as you (@SyazwanSamad ) already mentioned....just shift the force-displacement curve vertically, to have a nonzero force at displacement of zero. 
You can find a simple example here: [Components/Elements/Spring_Type13_with_prestress_and_failure](https://github.com/OpenRadioss/ModelExchange/tree/main/Components/Elements/Spring_Type13_with_prestress_and_failure) 

For other element types, we have the /INIBRI/ keyword in the manual, e.g. : https://help.altair.com/hwsolvers/rad/topics/solvers/rad/inibri_stress_starter_r.htm

Hope it helps you ?

Thanks and best,
Marian
